int32 track_id
float32[] keypoints  # [x1, y1, confidence1, x2, y2, confidence2, ..., xn, yn, confidencen]
